ackhandler/sent_packet_handler.go

```diff
@@ -380,6 +407,16 @@ func (h *sentPacketHandler) queuePacketForRetransmission(packetElement *PacketEl
 	h.stopWaitingManager.QueuedRetransmissionForPacketNumber(packet.PacketNumber)
 }
 
+func (h *sentPacketHandler) computeHandshakeTimeout() time.Duration {
+	duration := 2 * h.rttStats.SmoothedRTT()
+	if duration == 0 {
+		duration = 2 * defaultInitialRTT
+	}
+	duration = utils.MaxDuration(duration, minTPLTimeout)
+	// exponential backoff
+	return duration << h.handshakeCount
```

Comment:
Add a comment that there's an implicit max for the handshakeCount due to the timeout.

Reply:
What do you mean? I don't see an implicit max anywhere.

Reply:
The handshake timeout results in an implicit max value. Otherwise, we'd have to worry about handshakeCount > 64.